lua/rustaceanvim/commands/code_action_group.lua

```diff
@@ -287,6 +287,7 @@ function M.cleanup()
     M.state.secondary.clear()
   end
 
+  ---@diagnostic disable-next-line
```

Comment:
issue: We shouldn't disable all diagnostics.
This should be

```suggestion
  ---@diagnostic disable-next-line missing-fields
```

Ditto for the other ones. You can see the diagnostic ID in the warning message, e.g:

```
[Warning] Missing required fields in type `rustaceanvim.PartitionedActions`: `grouped`, `ungrouped` (missing-fields)
```

ditto for the other comments.